java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/DeleteMenuController.java

```diff
@@ -0,0 +1,67 @@
+package com.xxAMIDOxx.xxSTACKSxx.menu.api.v1;
+
+import com.xxAMIDOxx.xxSTACKSxx.core.api.dto.ErrorResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.UUID;
+
+/**
+ * Controller for Deleting a menu
+ *
+ * @author ArathyKrishna
+ */
+@RequestMapping("/v1/menu/{id}")
+public interface DeleteMenuController {
+    @DeleteMapping(consumes = "application/json", produces = "application/json")
+    @Operation(
+            tags = "Menu",
+            summary = "Removes a Menu with all it's Categories and Items",
+            description = "Remove a menu from a restaurant",
+            operationId = "DeleteMenu",
+            responses = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            description = "Success",
+                            content = @Content(schema = @Schema(hidden = true))),
+                    @ApiResponse(
```

Comment:
204 is a successful response.

`HTTP DELETE
A successful response of DELETE requests SHOULD be HTTP response code 200 (OK) if the response includes an entity describing the status, 202 (Accepted) if the action has been queued, or 204 (No Content) if the action has been performed but the response does not include an entity.`

If there's no body in the response, then i think a 204 makes sense instead of the 200